lib/musicbrainz-api.ts

```diff
@@ -15,13 +15,11 @@ import { DigestAuth } from './digest-auth';
 import { RateLimiter } from './rate-limiter';
 import * as mb from './musicbrainz.types';
 
-import got, { Options } from 'got';
+import ky, {Options, ResponsePromise} from 'ky-universal';
```

Comment:
Just keep the code styke
```suggestion
import ky, { Options, ResponsePromise } from 'ky-universal';
```